tests/devices/unit_tests/test_attenuator.py

```diff
@@ -0,0 +1,34 @@
+from unittest.mock import MagicMock
+
+import pytest
+from ophyd.sim import make_fake_device
+from ophyd.status import Status
+
+from dodal.devices.attenuator.attenuator import Attenuator
+
+CALCULATED_VALUE = 10
```

Comment:
Could: I would use a range or something here, just so that we have some variation that we can test against